src/test/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/entities/AsylumCaseTest.java

```diff
@@ -1,810 +0,0 @@
-package uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities;
```

Comment:
Has this test class been missed?  Should have one for the notifications-api too.

Reply:
done